combat-trainer.lic

```diff
@@ -22,6 +22,8 @@ class SetupProcess
     echo("  @cycle_armors: #{@cycle_armors}") if $debug_mode_ct
     @cycle_armors_time = settings.cycle_armors_time
     echo("  @cycle_armors_time: #{@cycle_armors_time}") if $debug_mode_ct
+    @armor_hysteresis = settings.armor_cycle_hysteresis || false
```

Comment:
❓ The name of this setting implies that the user would use it to turn the hysteresis feature on or off. However, its value is overwritten. What is the intent of this variable?